src/pages/steps/AdditionalInformationStep/AdditionalInformationStep.tsx

```diff
@@ -136,6 +139,12 @@ type Props = StackProps & {
   variant?: Values<typeof AdditionalInformationStepVariant>;
 };
 
+export enum ValidationStatus {
+  NONE = 'none',
+  WARNING = 'warning',
+  SUCCESS = 'success',
+}
```

Comment:
```suggestion
import { Values } from '@/types/Values';

export const ValidationStatus = {
  NONE: 'none',
  WARNING: 'warning',
  SUCCESS: 'success',
} as const;

export type ValidationStatus = Values<typeof ValidationStatus>;
```
We prefer using javascript objects and infering the type from it like this. See reasons why enums are not the best construct in typescript: https://www.youtube.com/watch?v=jjMbPt_H3RQ

And most of the time we prefer exporting the variables at the end of the file, so we can get a quick overview of what's exported.